src/modules/invocation/domain/invocation.domain.ts

```diff
@@ -43,4 +43,10 @@ export class Invocation extends Base {
     this.id = id;
     this.selectedMethod = selectedMethod;
   }
+
+  getContractIdValue = (inputString: string): string => {
+    const regex = /{{(.*?)}}/g;
```

Comment:
add `//comment` what this regex do

Reply:
The test I created in invocation.e2e.spec.ts demonstrates this functionality